CODEOWNERS

```diff
@@ -4,4 +4,4 @@
 #
 # More on CODEOWNERS files: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 
-* @hashicorp/vault-ecosystem-foundations
+* @hashicorp/vault-ecosystem
```

Comment:
This is showing as an error but I see the team here: https://github.com/orgs/hashicorp/teams/vault-ecosystem

Reply:
Should be fixed now. The team did not have perms on this repo.